grails-app/assets/javascripts/organisation.js

```diff
@@ -15,6 +17,233 @@ OrganisationViewModel = function (props, options) {
 
     var config = _.extend({}, defaults, options);
 
```

Comment:
The OrganisationPageViewModel appears to be a copy/paste of the OrganisationViewModel, leaving a large amount of code duplication.

Reply:
has been tidied up